### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by robertorodriguez12**
Added tech stack, adjusted images so they are displayed correctly. (They were showing as broken links/images before). Add a brief synopsis of what we did on this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,33 @@
 - Written in ES6+ using Babel + Webpack
 - Designed using Adobe Experience Design. Preview it [here](https://xd.adobe.com/view/a662a49f-57e7-4ffd-91bd-080b150b0317/).
 
+## Updates made by our team:
+* We are a small group (7 individuals) of Turing Students, for our current project, we were tasked with adding a style survey for users that will help them determine what their dress style is. The survey consists of a series of categories such as tops, bottoms, dresses, shoes, accessories, etc. After I have completed the survey, I am given a summary of my style such as ‘classic but edgy’. Once the user completes the style survey, their user object will be updated with a new column of style, this will allow us to filter the results that the user receives in order to more easily present items that would fit with their style. 
+
```
Comment (npdarrington) [inline README.md]: Let's adjust the series of categories sentence to the only 4 categories we have in our survey currently (tops, bottoms, shoes, accessories), this way it is not creating a false-positive for something that the survey will not perform yet. What do you think?

**Review by npdarrington — COMMENTED** [review]
Overall looks great! Please consider the in-line adjustment suggested, otherwise this looks great!

File: README.md (hunk 1 above)
Comment (robertorodriguez12) [reply README.md]: I like the way you think!

**Review by npdarrington — APPROVED** [review]
Looks good to me!

File: README.md [hunk 2]
```diff
@@ -9,6 +9,33 @@
 - Written in ES6+ using Babel + Webpack
 - Designed using Adobe Experience Design. Preview it [here](https://xd.adobe.com/view/a662a49f-57e7-4ffd-91bd-080b150b0317/).
 
+## Updates made by our team:
+* We are a small group (7 individuals) of Turing Students, for our current project, we were tasked with adding a style survey for users that will help them determine what their dress style is. The survey consists of a series of categories such as tops, bottoms, shoes and accessories. After I have completed the survey, I am given a summary of my style such as ‘classic but edgy’. Once the user completes the style survey, their user object will be updated with a new column of style, this will allow us to filter the results that the user receives in order to more easily present items that would fit with their style.
```
Comment (dhalverson) [inline README.md:13]: Not a big deal, but this should say 6 individuals

**Review by dhalverson — APPROVED** [review]
Looks good, thank you for putting this together!

**Pull request merged**